alerter.js

```diff
@@ -1,3 +1,5 @@
+const { expect } = require("chai");
+
 let alertFailureCount = 0;
 
 function networkAlertStub(celcius) {

```

Comment:
With the assumption that the production code is modified to pass the test, following are my comments...